commands/active.go

```diff
@@ -21,3 +27,38 @@ func cmdActive(c *cli.Context) {
 		fmt.Println(host.Name)
 	}
 }
+
+func getActiveHost(store persist.Store) (*host.Host, error) {
+	hosts, err := listHosts(store)
+	if err != nil {
+		return nil, err
+	}
+
+	hostListItems := getHostListItems(hosts)
+
+	for _, item := range hostListItems {
+		if item.Active {
+			h, err := loadHost(store, item.Name)
+			if err != nil {
+				return nil, err
+			}
+			return h, nil
+		}
+	}
+
+	return nil, errors.New("Active host not found")
+}
+
+// IsActive provides a single function for determining if a host is active
+// based on both the url and if the host is stopped.
+func isActive(h *host.Host, currentState state.State, url string) (bool, error) {
```

Comment:
Yeah, code in `ls.go` requires this function, although it looks like they should both use the same code if possible.  It may require some re-jiggering to get there, though